lib/Doctrine/DBAL/Types/Type.php

```diff
@@ -36,6 +36,7 @@
     const TARRAY = 'array';
     const SIMPLE_ARRAY = 'simple_array';
     const JSON_ARRAY = 'json_array';
+    const JSON_UNFLAWED = 'json';
```

Comment:
this should just be ``JSON``, not ``JSON_UNFLAWED``. Type name constant are matching the type names (unless it is not possible because of reserved keywords like ``array``)